### PR TITLE
Add Redis install/uninstall scripts

### PR DESCRIPTION
**Pull request by J-Sek**
### Step template checklist

- [x] `Id` should be a **GUID** that is not `00000000-0000-0000-0000-000000000000`
  - **NOTE** If you are modifying an existing step template, please make sure that you **do not** modify the `Id` property *(updating the `Id` will break the Library sync functionality in Octopus)*. 
- [ ] `Version` should be incremented, otherwise the integration with Octopus won't update the step template correctly
- [x] Parameter names should not start with `$`
- [x] **To minimize the risk of step template parameters clashing with other variables in a project that uses the step template, ensure that you prefix your parameter names (e.g. an abbreviated name for the step template or the category of the step template**
- [x] If a new `Category` has been created:
   - [x] An image with the name `{categoryname}.png` must be present under the `step-templates/logos` folder
   - [ ] The `switch` in the `humanize` function in [`gulpfile.babel.js`](https://github.com/OctopusDeploy/Library/blob/master/gulpfile.babel.js#L92) must have a `case` statement corresponding to it
- [x] `LastModifiedBy` field must be present, and (_optionally_) updated with the correct author

### CODE REVIEW

File: step-templates/redis-install.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    "Octopus.Action.Script.ScriptFileName": null,
+    "Octopus.Action.Package.FeedId": null,
+    "Octopus.Action.Package.PackageId": null
+  },
+  "Parameters": [
+    {
+      "Id": "2fdb42bf-b1f0-4187-b925-bdf4975090e4",
+      "Name": "Redis Unpackage step",
```
Comment (alfhenrik) [inline step-templates/redis-install.json]: Could you please remove the spaces in the parameter names and prefix with `ris_`

File: step-templates/redis-install.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
+{
+  "Id": "3cf1dadf-169d-4fd2-b95b-f5f07343b911",
+  "Name": "Redis - Install service",
+  "Description": "Script will make sure that Redis server is installed as a Windows service\n\n- It will start Redis inside package directory\n- To provide custom config you can prepare your own package (e.g. my-redis-64) or improve this template",
```
Comment (alfhenrik) [inline step-templates/redis-install.json]: This `Throw "$outputPath did not found in previous step. Make sure you point to 'Deploy package' type of step"` needs a bit of rewording, perhaps `Unable to find output path for step $step. Make sure you've selected the correct step for your package.`

File: step-templates/redis-uninstall.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    "Octopus.Action.RunOnServer": "false",
+    "Octopus.Action.Script.ScriptBody": "$name = $OctopusParameters['Redis Service name']\r\n\r\n$service = Get-Service -Name $name -ErrorAction SilentlyContinue\r\n\r\nif (-not $service) {\r\n    Write-Host \">>> $name service not found. Skipping this time.\"\r\n    return\r\n}\r\n\r\nWrite-Host \">>> Uninstalling with: sc.exe\"\r\nif ($service.Status -eq 'Running') {\r\n    &\"sc.exe\" stop $name | Write-Host\r\n}\r\n&\"sc.exe\" delete $name | Write-Host\r\n\r\n$limit = 15\r\nwhile (Get-Service -Name $name -ErrorAction SilentlyContinue) {\r\n    Start-Sleep -s 1\r\n    \r\n    $limit = $limit - 1\r\n    if ($limit -eq 0) {\r\n        Throw \"Unable to stop Redis service within 15s\"\r\n    }\r\n}",
+    "Octopus.Action.Script.ScriptFileName": null,
+    "Octopus.Action.Package.FeedId": null,
+    "Octopus.Action.Package.PackageId": null
+  },
+  "Parameters": [
+    {
+      "Id": "12713171-c487-4118-8a4c-7bd8c2e5e269",
+      "Name": "Redis Service name",
```
Comment (alfhenrik) [inline step-templates/redis-uninstall.json]: Could you please remove the spaces in the parameter names and prefix with `rus_`

File: step-templates/redis-install.json (hunk 2 above)
Comment (J-Sek) [reply step-templates/redis-install.json]: It was too late at night :P Thanks for pointing this one out.

File: step-templates/redis-uninstall.json (hunk 3 above)
Comment (J-Sek) [reply step-templates/redis-uninstall.json]: Done. I was not sure if this is really required since existing templates rarely conform to this rule.

Comment (alfhenrik) [reply step-templates/redis-uninstall.json]: Correct, this is a pretty new requirement from our side to limit the risk of duplicate parameters between step templates and variables in projects/library variables sets

Comment (alfhenrik) [reply step-templates/redis-uninstall.json]: Looks like this one still needs to be updated with prefix and removal of spaces

Comment (J-Sek) [reply step-templates/redis-uninstall.json]: Apologies, I had hit commit before saving the file. Now its updated

**Pull request merged**